arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -238,6 +238,8 @@ RocksDBEngine::RocksDBEngine(application_features::ApplicationServer& server)
       _dbExisted(false),
       _runningRebuilds(0),
       _runningCompactions(0),
+      _autoFlushCheckInterval(60.0 * 60.0),
+      _autoFlushMinWalFiles(10),
```

Comment:
Yes. We want to avoid too frequent flushes, and especially not flushes that do not provide any benefit.
If we have < 10 live journal files, these will not occupy a lot of disk storage.
The problem we are trying to solve with this PR is RocksDB having lots of live WAL files open from previous runs, which in total can consume GBs to TBs.